### PR TITLE
add build version, and plugin hash.

### PR DESCRIPTION
**Pull request by vwxyzh**
@chenkennt @superyyrrzz @hellosnow @ansyral @qinezh 
#449

### CODE REVIEW

File: src/Microsoft.DocAsCode.Build.Engine/DocumentBuilder.cs [hunk 1]
```diff
@@ -856,6 +864,24 @@ private IMarkdownService CreateMarkdownService(DocumentBuildParameters parameter
                 });
         }
 
+        private static string ComputePluginHash(List<Assembly> assemblyList)
+        {
+            using (var ms = new MemoryStream())
+            using (var writer = new StreamWriter(ms))
+            {
+                foreach (var item in from ass in assemblyList
```
Comment (qinezh) [inline src/Microsoft.DocAsCode.Build.Engine/DocumentBuilder.cs]: ass => assembly?

**Comment by qinezh** [comment]
:+1:

File: src/Microsoft.DocAsCode.Build.Engine/DocumentBuilder.cs (hunk 1 above)
Comment (hellosnow) [inline src/Microsoft.DocAsCode.Build.Engine/DocumentBuilder.cs]: >assemblyList [](start = 49, length = 12)

assemblyList may be null?

**Comment by hellosnow** [comment]
:shipit:

**Pull request merged**